evm/client/chain_client.go

```diff
@@ -102,7 +101,7 @@ type chainClient struct {
 		*big.Int,
 		*RPCClient,
 	]
-	txSender     *multinode.TransactionSender[*types.Transaction, *SendTxResult, *big.Int, *RPCClient]
+	txSender     *multinode.TransactionSender[*types.Transaction, struct{}, *big.Int, *RPCClient]
```

Comment:
evm doesn't even have a separate result type, so lots of boilerplate can be dropped

Reply:
Generic SendTxResult was added to handle cases when some chains do not have return parameters except error. 
Is not it dirtier to define struct{} as result instead of implementing an interface?

Reply:
> Is not it dirtier to define struct{} as result instead of implementing an interface?

I think the net effect of this set of changes speaks for itself :shrug: 
The no-op case (this one) was burdened with dozens of lines of extra boilerplate just in order to opt-out of a feature.